apps/web/src/client/components/file-uploader/file-uploader.module.js

```diff
@@ -0,0 +1,9 @@
+import registerUIEvents from './_ui.js';
+
+const allUploadForms = document.querySelectorAll('.pins-file-upload');
+
+const initFileUploaderModule = () => {
+	registerUIEvents(allUploadForms[0]);
```

Comment:
I would do this check `if (!uploadForm.length) return;` here

Reply:
it's in the first line of the function registerUIEvents:

<img width="373" alt="image" src="https://user-images.githubusercontent.com/101253567/199473128-b59be74f-398d-47e4-875d-f39e4009f541.png">

Reply:
yes - I'd move it there so we can establish a pattern for future components also